packages/date-time/src/components/Calendar/examples/Calendar.Inline.Example.tsx

```diff
@@ -38,6 +38,7 @@ export interface ICalendarInlineExampleProps {
   showWeekNumbers?: boolean;
```

Comment:
line 5/6 in this file should probably use the full package import (examples should avoid using relative import)

Reply:
this is actually causing an issue. If I change it to reference "@uifabric/date-time/lib/components/Calendar", the build fails with a "file does not exist". I'll add a work item for us to fix this later, but I'll revert the change for now.

Reply:
opened #8146 for us to look in to